priv/templates/phx.gen.auth/registration_live.ex

```diff
@@ -0,0 +1,74 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>RegistrationLive do
+  use <%= inspect context.web_module %>, :live_view
+
+  alias <%= inspect context.module %>
+  alias <%= inspect schema.module %>
+
+  def render(assigns) do
+    ~H"""
+    <h1>Register</h1>
+
+    <.form
+      id="registration_form"
+      :let={f}
+      for={@changeset}
+      phx-submit="save"
+      phx-change="validate"
+      phx-trigger-action={@trigger_submit}
+      action={Routes.<%= schema.route_helper %>_session_path(@socket, :create, %{_action: "registered"})}
```

Comment:
Chris explained to me that the form has the `put` method because we're saving the user into the db before triggering the controller action. The form uses the put method because it assumes that an update will be done.